IncludeToolbox/IncludeFormatter/IncludeComparer.cs

```diff
@@ -12,14 +12,14 @@ public IncludeComparer(string[] precedenceRegexes, string documentName)
         {
             string currentFilename = documentName.Substring(0, documentName.LastIndexOf('.'));
 
-            this.precedenceRegexes = new string[precedenceRegexes.Length];
-            for (int i = 0; i < this.precedenceRegexes.Length; ++i)
+            PrecedenceRegexes = new string[precedenceRegexes.Length];
+            for (int i = 0; i < PrecedenceRegexes.Length; ++i)
             {
-                this.precedenceRegexes[i] = precedenceRegexes[i].Replace(CurrentFileNameKey, currentFilename);
+                PrecedenceRegexes[i] = precedenceRegexes[i].Replace(CurrentFileNameKey, currentFilename);
             }
         }
 
-        private readonly string[] precedenceRegexes;
+        public string[] PrecedenceRegexes { get; set; }
```

Comment:
Because this does the current filename replace, I needed to get the fixed up regexes.